packages.yml

```diff
@@ -2,4 +2,4 @@ packages:
   - package: dbt-labs/dbt_utils
     version: [">=0.9.2","<1.0.0"]
   - package: tuva-health/the_tuva_project
-    version: [">=0.5.0","<1.0.0"]
```

Comment:
This version range looks wrong. I think it should be [">=0.6.0","<0.7.0"]

Reply:
@sarah-tuva this comment is on the 'before' diff.  The package version has been correctly updated.